src/stories/Quiz/tests/mocks.ts

```diff
@@ -205,6 +205,10 @@ export const useMockContextValue = (question?: Question): QuizContextValue => {
       ...mockElementProps,
       className: 'cio-question-cta-button',
     }),
+    getSkipQuestionButtonProps: () => ({
+      ...mockElementProps,
+      className: 'cio-question-cta-button',
```

Comment:
Should this be `cio-question-skip-button` or does it not matter?

Reply:
[f18aa78](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/132/commits/f18aa78821419475ac9b701bcd44f4f165819fca)